packages/eslint-config/eslint.config.mjs

```diff
@@ -0,0 +1,115 @@
+import js from '@eslint/js';
+import json from '@eslint/json';
+import eslintConfigPrettier from 'eslint-config-prettier';
+import eslintPluginPerfectionist from 'eslint-plugin-perfectionist';
+import eslintPluginReact from 'eslint-plugin-react';
+import globals from 'globals';
+import tseslint from 'typescript-eslint';
+//import nlDesignSystemJS from './rules/js.mjs';
+//import nlDesignSystemReact from './rules/react.mjs';
```

Comment:
Ik  heb deze twee er vooralsnog uitgelaten, ik ben door beide set met rules heengegaan en de deprecated rules of rules die ook al in @eslint/js/recommended zitten heb ik verwijderd.

Hoe graag willen we deze rules nog gebruiken?

Reply:
js.mjs mag bij de default config erbij, react mag eruit